agir/groups/serializers.py

```diff
@@ -88,15 +88,17 @@ class SupportGroupSerializer(FlexibleFieldsMixin, serializers.Serializer):
     isFull = serializers.SerializerMethodField(read_only=True)
 
     routes = RoutesField(routes=GROUP_ROUTES, read_only=True)
-    isCertified = serializers.BooleanField(read_only=True, source="is_certified")
+    isCertified = serializers.SerializerMethodField(read_only=True)
 
     def to_representation(self, instance):
         user = self.context["request"].user
+
         self.membership = None
         if not user.is_anonymous and user.person:
-            self.membership = instance.memberships.filter(
-                person_id=user.person.id
-            ).first()
+            for membership in instance.memberships.all():
+                if membership.person_id == user.person.id:
+                    self.membership = membership
```

Comment:
C'est vraiment plus rapide qu'avec le `filter` dans la requête `instance.memberships` ?
```suggestion
            for membership in instance.memberships.all():
                if membership.person_id == user.person.id:
                    self.membership = membership
                    break
```